packages/cli/src/WorkflowHelpers.ts

```diff
@@ -703,3 +704,68 @@ export function generateFailedExecutionFromError(
 		stoppedAt: new Date(),
 	};
 }
+
+/** * Get all nodes in a workflow where the node credential is not accessible to the user. */
```

Comment:
```suggestion
/** Get all nodes in a workflow where the node credential is not accessible to the user. */
```
Else it's parsed as a bullet point:
<img width="760" alt="image" src="https://user-images.githubusercontent.com/44588767/197550633-8a8e173b-2139-473d-bcd6-5e7003be5e9c.png">